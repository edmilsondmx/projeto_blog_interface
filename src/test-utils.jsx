import React from "react";
import { BrowserRouter } from "react-router-dom";
import ThemeProvider from "./styles/ThemeProvider";
import { render } from "@testing-library/react";

const TestProvider = ({ children }) => (
  <BrowserRouter basename="/">
    <ThemeProvider>{children}</ThemeProvider>
  </BrowserRouter>
);

export * from "@testing-library/react";

const customRender = (ui, options) =>
  render(ui, { wrapper: TestProvider, ...options });

export { customRender as render };
