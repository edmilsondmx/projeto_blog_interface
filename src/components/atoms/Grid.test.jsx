import React from "react";
import { render } from "test-utils";
import Grid from "./Grid";

it("renders without crashing", () => {
  render(<Grid />);
});
