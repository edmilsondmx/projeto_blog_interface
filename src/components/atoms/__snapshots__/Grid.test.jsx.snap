// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-beqWaB dLcSoP"
  />
</DocumentFragment>
`;

exports[`match snapshot with params 1`] = `
<DocumentFragment>
  <div
    class="sc-beqWaB iCQipX"
  />
</DocumentFragment>
`;

exports[`match snapshot with sm an lg 1`] = `
<DocumentFragment>
  <div
    class="sc-beqWaB sNqNg"
  />
</DocumentFragment>
`;
